Backend/app/dashboard/CommonAPIs/spec/ProviderPlatform/Management/API/Payout.yaml

```
@@ -170,3 +194,13 @@
    - driverIds: [Id Driver]
    - blockState: Bool
    - derive: "'HideSecrets"
+  PayoutVpaStatus:
+    - enum: "VIA_WEBHOOK,MANUALLY_ADDED,VERIFIED_BY_USER"
+  UpdateVpaReq:
+   - driverId: Id Driver
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 3

(indentation)